### PR TITLE
Change feed processor options - Added option to configure StartTime

### PR DESCRIPTION
**Pull request by mariusz-pawlowski**
We need to give option to configure StartDateTime.

* to keep backward compatibilitiy, when StartDateTime isn't be defined, then DateTime.Min will be set
* there are some databases where data are not good quality (i.e junk data on DEV), and if we start processing then exceptions are thrown

### CODE REVIEW

**Comment by mariusz-pawlowski** [comment]
@LarsSkovslund  will you take a look on this?

**Review by LarsSkovslund — CHANGES_REQUESTED** [review]

File: src/Atc.Cosmos/ChangeFeedProcessorOptions.cs [hunk 1]
```diff
@@ -29,9 +29,9 @@ public class ChangeFeedProcessorOptions
         /// This is only used when:
         /// (1) Lease store is not initialized and is ignored if a lease exists and has continuation token.
         /// (2) StartContinuation is not specified.
-        /// If not specified, then default value will be taken DateTime.Min.
+        /// Default value is DateTime.Min.
         /// </remarks>
-        public DateTime? StartTime { get; set; }
+        public DateTime StartTime { get; set; } = DateTime.MinValue.ToUniversalTime();
```
Comment (LarsSkovslund) [inline src/Atc.Cosmos/ChangeFeedProcessorOptions.cs:34]: Would you also please change the property name from `StartTime` to `DefaultStartTime`

File: src/Atc.Cosmos/ChangeFeedProcessorOptions.cs [hunk 2]
```diff
@@ -22,6 +22,17 @@ public class ChangeFeedProcessorOptions
...
+        /// <summary>
+        /// Gets or sets the time (exclusive) to start looking for changes after.
+        /// </summary>
+        /// <remarks>
+        /// This is only used when:
+        /// (1) Lease store is not initialized and is ignored if a lease exists and has continuation token.
+        /// (2) StartContinuation is not specified.
+        /// If not specified, then default value will be taken DateTime.Min.
+        /// </remarks>
+        public DateTime? StartTime { get; set; }
```
[the remark this answers is not in the document]
Comment (mariusz-pawlowski) [reply src/Atc.Cosmos/ChangeFeedProcessorOptions.cs]: Fix has been done - I've moved Default value to ChangeFeedProcessorOptions.StartTime definition.
Property description (summary) was inspired with official azure cosmos sdk documentation so is this time good enough?

[the remark this answers is not in the document]
Comment (LarsSkovslund) [reply src/Atc.Cosmos/ChangeFeedProcessorOptions.cs]: You are right, lets not confuse existing users of Cosmos SDK by using another name.

**Pull request merged**